plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -47,6 +47,8 @@ function arrayStringifyHelper(arr: Array<unknown>): string {
   return '[' + arr.toString() + ']';
 }
 
+export type SqlCommenterAttributes = Record<string, string>;
```

Comment:
Would it be preferred that I list all the known attributes instead of the index signature? Or both?